packages/compat/src/compat-app-builder.ts

```diff
@@ -1277,11 +1277,16 @@ export class CompatAppBuilder {
     let amdModules = nonFastboot.map(file => this.importPaths(appFiles, file));
     let fastbootOnlyAmdModules = fastboot.map(file => this.importPaths(appFiles, file));
 
-    // this is a backward-compatibility feature: addons can force inclusion of
-    // modules.
-    eagerModules.push('./-embroider-implicit-modules.js');
-
-    let params = { amdModules, fastbootOnlyAmdModules, lazyRoutes, lazyEngines, eagerModules, styles };
+    let params = {
+      amdModules,
+      fastbootOnlyAmdModules,
+      lazyRoutes,
+      lazyEngines,
+      eagerModules,
+      styles,
+      // this is a backward-compatibility feature: addons can force inclusion of modules.
```

Comment:
thanks for the comment here!

Reply:
😂 the comment will need to be removed when this file gets coopted to do more stuff, this comment was just brought forward from the bit it was deleted from just above this 👍